draft-ietf-rtcweb-jsep.xml

```diff
@@ -401,10 +401,58 @@ setRemote(ANSWER) |                                   |
         <t>Note that most applications should be able to treat the
         SessionDescriptions produced and consumed by these various API calls
         as opaque blobs; that is, the application will not need to read or
-        change them. The W3C WebRTC API specification will provide appropriate
-        APIs to allow the application to control various session parameters,
-        which will provide the necessary information to the browser about what
-        sort of SessionDescription to produce.</t>
+        change them.</t>
+      </section>
+
+      <section title="Session Description Control" anchor="sec.session-description-ctrl">
+	<t>In order to give the application control over various
+	common session parameters, JSEP provides control surfaces
+	which tell the browser how to generate session
+	descriptions. This avoids the need for manual modification of
+	session descriptions in most cases.</t>
+
+	<t>Changes to these objects are realized in the session
+	descriptions generated by subsequent createOffer/Answer
+	calls.</t>
+
+	<section title="RtpTransceivers" anchor="sec.rtptransceivers">
+	  <t>RtpTransceviers generally have a 1:1 mapping with m=
+	  sections, however there may be more RtpTransceivers than m=
+	  sections when RtpTransceivers are created but not yet
+	  associated with an m= section, or if RtpTransceivers have
+	  been stopped and disassociated from m= sections. An
+	  RtpTransceiver is never associated with more than one m=
+	  section, and an m= section is always associated with exactly
+	  one RtpTransceiver once a session description is
+	  applied.</t>
```

Comment:
Do you mean stopped by a remote offer?  That's an interested question of what happens to the transceiver, because it can be rolled back.  Perhaps we should delay any effect on the transceiver until the answer is set in the local description.  I have a question for you on the whiteboard that I'd like to get your input on.